### PR TITLE
[Sound] Deprecate activate concept and add running concept

### PR DESCRIPTION
**Pull request by pkosko**
[ACR] https://code.sec.samsung.net/jira/browse/TWDAPI-288

### CODE REVIEW

**Review by omar-srbd — COMMENTED** [review]
Hello @pkosko, after a short analysis of the files in this PR, I have noticed that the method '**getActivatedDeviceList()**' has multiple mentions in the _application/web/guides/multimedia/audio_ file of the Tizen docs repo.
Should this mention of **getActivatedDeviceList()** be removed/changed?

**Comment by pkosko** [comment]
> Hello @pkosko, after a short analysis of the files in this PR, I have noticed that the method '**getActivatedDeviceList()**' has multiple mentions in the _application/web/guides/multimedia/audio_ file of the Tizen docs repo. Should this mention of **getActivatedDeviceList()** be removed/changed?

thank you, you are right. I've updated a guide.

**Review by omar-srbd — APPROVED** [review]
@pkosko, thanks for making the thanks on **.md** file, all other changes in this PR seems alright to me.

@asm-foysal, I am approving this PR for merging.

File: docs/application/web/guides/multimedia/audio.md [hunk 1]
```diff
@@ -96,13 +96,15 @@ Learning how to list connected and activated sound devices allows you to manage
    }
    ```
 
-2. Get a list of the current sound devices in an activated state using the `getActivatedDeviceList()` method:
+2. Get a list of the current sound devices in an running state using the `getConnectedDeviceList()` method and checking isRunning property:
```
Comment (omar-srbd) [inline docs/application/web/guides/multimedia/audio.md:99]:
```suggestion
2. Get a list of the current sound devices in a running state using the `getConnectedDeviceList()` method and checking the isRunning property:
```

**Review by omar-srbd — COMMENTED** [review]
@pkosko, please check the correction suggestion and implement the change.
Thank you.

**Review by omar-srbd — APPROVED** [review]
@pkosko, thanks for making the suggested change.

@asm-foysal, I am approving this PR for merging.

**Pull request merged**